invenio_app_ils/cli.py

```diff
@@ -157,6 +156,11 @@ def _get_valid_status(item, items_on_loans):
             "request_expire_date": expire_date.strftime("%Y-%m-%d"),
         }
 
+        if status == "PENDING":
+            loan[Document.pid_field] = "{}".format(item[Document.pid_field])
```

Comment:
We need the `Document.pid_field` in every case not only when its `PENDING`

Reply:
Yes but right now is not supported in invenio-circulation so to be consistent I din't include it here. I created a ticket about that to handle it https://github.com/inveniosoftware/invenio-circulation/issues/85

Reply:
👍 for this one, I am about to address also the issue of the demo loan data and how they reflect reality. We need this one.